src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.targets

```diff
@@ -3,7 +3,11 @@
     Capability that enables Visual Studio support for hosting Asp.Net Core applications in the IIS process
   -->
   <ItemGroup>
-    <ProjectCapability Include="AspNetInProcessHosting" />
```

Comment:
So, does this mean we can drop the AspNetInProcessHosting capability? I think this was added to the 2.1 Microsoft.AspNetCore.App metapackage and wasn't removed when we yanked in-proc.

Reply:
Yeah we forgot to drop it from 2.1 .App. We will have to rename it, hence AspNetCoreInProcessHosting.
@natemcmaster is there a way to make M.A.App transitively get the project capability from .IIS or .IISIntegration?

Reply:
It could come transitively from .IISIntegration, but not .IIS.

Reply:
Why is that? Because .IIS is not part of .App yet? I think the plan is to add it to .App for 2.2

Reply:
.IIS can't be part of .App because it contains Windows-only binaries. It can be part of runtime.win.Microsoft.AspNetCore.App, but not the portable, "Microsoft.AspNetCore.App" version.

Reply:
Then we will need to add it to .App as we can't require .IIS to include .IISIntegration

Reply:
Ok. Here's the cleanup: https://github.com/aspnet/Universe/pull/1237